local/jimm/add-controller.sh

```diff
@@ -26,6 +26,7 @@ if [ $jimmctlAvailable -ne 0 ]; then
     echo "Building jimmctl..."
     # Build jimmctl so we may add a controller.
     go build ./cmd/jimmctl
+    sudo ln -s $PWD/jimmctl /usr/local/bin/jimmctl
```

Comment:
Not sure if this is the right way to make the binary available, but with out this the following calls to `jimmctl` (and the `which jimmctl` above) don't use the local `./jimmctl`

Reply:
Ah is this in the case of the snap being available? If so, coudl you update this PR to do "./" infront of all jimmctl calls?

Reply:
This is actually the inverse, without this fix it will build jimmctl but then it'll display an error about jimmctl not existing as it's trying to use the global command.

However, if you prefer we could symlink the snap jimmctl into the project directory if it exists and then use `./jimmctl` which wouldn't pollute the global bin.

Reply:
Actually, symlinking the snap won't work as then the `which jimmctl` lines won't work.

Reply:
I think just doing ./ before each jimmctl is fine and clearer

Reply:
> I think just doing ./ before each jimmctl is fine and clearer

This would mean it would always use the locally built jimmctl and not the jimmctl from the snap (if available) so if we switch to `./jimmctl` I suppose we should also remove the other parts of the script that handle the snap jimmctl.

Reply:
I've removed the snap handling from the script.

Reply:
I'm ooo, but saw this come through in an email. Don't remove the snap handling because the Github Action in .github/actions/test-server spins up a JIMM test env and doesn't require Go to be installed, rather opting to use the jimmctl snap and uses this add-controller script.

Reply:
I suggest either using the original symlink idea or making a bash variable that becomes either "./" or empty string and placing it in front of calls to jimmctl.

Reply:
Thanks @kian99 I've undone my changes to remove the snap handling and instead this now checks for the existence of `./jimmctl` instead of just relying on `which jimmctl` and swaps out the jimmctl command (stored in a variable).